smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/annotation-trait-extra-properties.errors

```diff
@@ -1 +1 @@
-[WARNING] ns.foo#Baz: Error validating trait `sensitive`: Invalid structure member `test` found for `smithy.api#sensitive` | TraitValue
+[WARNING] ns.foo#Baz: Error validating trait `sensitive`: Invalid structure member `test` found for `smithy.api#sensitive` | TraitValue.smithy.api#sensitive.test
```

Comment:
I'm not sure how useful this one is -- do you have a use case for suppressing this? How does pathing work here for nested things? Can you just include the shape ID and omit the path?

Reply:
Well we need the path because one trait could have multiple invalid structure members, each as a different violation